pkg/fixer/fixes/nonrawregexpattern_test.go

```diff
@@ -0,0 +1,229 @@
+package fixes
+
+import (
+	"testing"
+
+	"github.com/styrainc/regal/pkg/report"
+)
+
+func TestNonRawRegexPattern(t *testing.T) {
+	t.Parallel()
+
+	testCases := map[string]struct {
+		contentAfterFix string
+		fc              *FixCandidate
+		fixExpected     bool
+		runtimeOptions  *RuntimeOptions
+	}{
+		"none needed, no location": {
+			fc: &FixCandidate{Filename: "test.rego", Contents: `package test
+
+all_digits if {
+    regex.match(` + "`" + `[\\d]+` + "`" + `, "12345")
+}
+`},
+			contentAfterFix: `package test
+
+all_digits if {
+    regex.match(` + "`" + `[\\d]+` + "`" + `, "12345")
```

Comment:
So as per above comment, this should be:

```
`[\d]+`
```

Reply:
Ahh, very good spot. I have fixed this in 32c8f22b0d46a1ea2823bf6e1eb26f64becdb99a!